scio-bigquery/src/main/scala/com/spotify/scio/bigquery/types/ConverterProvider.scala

```diff
@@ -81,7 +83,7 @@ private[types] object ConverterProvider {
         case t if t =:= typeOf[Array[Byte]] =>
           val b = q"$tree.asInstanceOf[_root_.java.nio.ByteBuffer]"
           q"_root_.java.util.Arrays.copyOfRange($b.array(), $b.position(), $b.limit())"
-
```

Comment:
Nit: keep these empty lines to minimize unrelated style changes? Same for the next few cases of adding/removing blank lines.